api/src/main/java/openconsensus/metrics/LongCumulative.java

```diff
@@ -75,33 +76,12 @@
  * @since 0.1.0
  */
 @ThreadSafe
-public interface LongCumulative extends Metric {
+public interface LongCumulative extends Metric<TimeSeries> {
```

Comment:
Should this be `LongCumulative.TimeSeries`?

Reply:
I had an import in the file for LongCumulative.TimeSeries, but changed it to be consistent.